rules/CodingStyle/Rector/FuncCall/ConsistentPregDelimiterRector.php

```diff
@@ -232,9 +232,11 @@ private function refactorArgument(Arg $arg): void
             }
 
             // change delimiter
-            if (strlen($innerPattern) > 2 && $innerPattern[0] === $innerPattern[strlen($innerPattern) - 1]) {
+            if (strlen((string) $innerPattern) > 2 && $innerPattern[0] === $innerPattern[strlen(
```

Comment:
It seems got `PHPStan\Type\MixedType`:

```bash
PHPStan\Type\MixedType #103859
   subtractedType: null
   isExplicitMixed: false
```

Reply:
resolved by skip if its scope has parent scope https://github.com/rectorphp/rector-src/pull/1655/commits/6f5cd0c0fcf3fc1db78aace04b729b74c1912378

Reply:
check parent scope exists seems not the correct way, as it will cause in closure use skipped https://github.com/rectorphp/rector-src/pull/1655/commits/8b104169237c326dbb1bfbd38544c4c28eb68fbf : 

```php
        function () use ($subject) {
            preg_split('#a#', $subject);
        };
```

Reply:
on closure use, that's resolved with check instanceof `PHPStan\Type\ErrorType` https://github.com/rectorphp/rector-src/pull/1655/commits/e1619b757049e65287df197d3d4ffd7ee772daad